tests/unit/dojson/test_dojson_conferences.py

```diff
@@ -580,26 +582,35 @@ def test_series_number_from_411__n():
         '</datafield>'
     )  # record/1447029
 
-    expected = 7
     result = clean_record(conferences.do(create_record(snippet)))
 
-    assert expected == result['series_number']
+    expected = [
+        {
+            'number': 7
+        },
+    ]
+    assert result['series'] == expected
```

Comment:
@jacquerie is right. @tomaszgy Can you please amend your rules to not generate a lonely `number` when `name` is not available?